libs/core/src/lib/carousel/carousel-item/carousel-item.component.ts

```diff
@@ -64,7 +70,7 @@ export class CarouselItemComponent implements CarouselItemInterface {
 
```

Comment:
- Could you add missing `@hidden` annotations for `HostBindings`?
- This property:
```
    @HostBinding('class.fd-carousel__item')
    carouselItem = 'fd-carousel__item';
```
should be changed to:
```
    @HostBinding('class.fd-carousel__item')
    carouselItem = true;
```

Reply:
this is fixed now.